ansible/playbooks/start-redpanda.yml

```diff
@@ -45,6 +46,16 @@
         --ips {{ groups["redpanda"] | map('extract', hostvars) | map(attribute='private_ip') | join(',') }}
       {% endif %}
 
+      {% if rack is defined and rack != -1 %}
+      sudo rpk redpanda config set redpanda.rack {{ rack }}
+      {% endif %}
+
+  - name: enable rack awareness
+    shell: |
+      set -e
+      rpk cluster config set enable_rack_awareness true --api-urls {{ hostvars[inventory_hostname].private_ip }}:9644
```

Comment:
The `--api-urls` is a comma-separated list of admin API addresses (IP:port) but this will give you a single address, can we change this so we map each IP to a list (see line 46)?

[question], Should we add a conditional so we use the `inventory_hostname` if "`use_public_ips`" is set?

Reply:
@vuldin also addressed this